udapi/block/tokenize/onwhitespace.py

```diff
@@ -26,15 +26,15 @@ class OnWhitespace(Block):
 
     Parameters
     ----------
-    normalize_spaces : bool
-        preserve whitespaces by filling MISC attributes `SpacesAfter` and `SpacesBefore` (by default True)
+    keep_spaces : bool
+        preserve whitespaces by filling MISC attributes `SpacesAfter` and `SpacesBefore` (by default False)
     """
 
     escape_whitespace_table = str.maketrans({' ':r'\s', '\t':r'\t', '\r':r'\r', '\n':r'\n'})
 
-    def __init__(self, normalize_spaces=True, **kwargs):
+    def __init__(self, keep_spaces=True, **kwargs):
```

Comment:
Shouldn't the default be False, as written in the documentation?

Reply:
Yes, thanks.